generators/spring-controller/index.js

```diff
@@ -136,15 +136,25 @@ module.exports = class extends BaseGenerator {
                     this.log(chalk.green(`adding ${action.actionMethod} action '${action.actionName}' for /api/${this.apiPrefix}/${action.actionPath}`));
                 });
 
-                this.template(
-                    `${SERVER_MAIN_SRC_DIR}package/web/rest/Resource.java.ejs`,
-                    `${SERVER_MAIN_SRC_DIR}${this.packageFolder}/web/rest/${this.controllerClass}.java`
-                );
-
-                this.template(
-                    `${SERVER_TEST_SRC_DIR}package/web/rest/ResourceIntTest.java.ejs`,
-                    `${SERVER_TEST_SRC_DIR}${this.packageFolder}/web/rest/${this.controllerClass}IntTest.java`
-                );
+                if (useBlueprint) {
+                    this.template(
+                        `${this.fetchFromInstalledJHipster('spring-controller/templates')}/${SERVER_MAIN_SRC_DIR}package/web/rest/Resource.java.ejs`,
+                        `${SERVER_MAIN_SRC_DIR}${this.packageFolder}/web/rest/${this.controllerClass}.java`
+                    );
+                    this.template(
+                        `${this.fetchFromInstalledJHipster('spring-controller/templates')}/${SERVER_TEST_SRC_DIR}package/web/rest/ResourceIntTest.java.ejs`,
+                        `${SERVER_TEST_SRC_DIR}${this.packageFolder}/web/rest/${this.controllerClass}IntTest.java`
+                    );
+                } else {
```

Comment:
@pascalgrimaud actually I think it will work without the else part as well, since with or without blueprint the `this.fetchFromInstalledJHipster` will provide the correct path to templates

Reply:
ok, we should give a try without

Reply:
you're right @deepu105 !
No need to keep the condition if/else

Reply:
awesome

Reply:
so lets merge the PR then